src/backends/gpu/ions.hpp

```diff
@@ -14,6 +14,12 @@ void nernst(std::size_t n, int valency,
             const fvm_value_type* Xi,
             fvm_value_type* eX);
 
+// prototype for nernst equation cacluation
```

Comment:
Comment doesn't match (cut-and-paste error by the looks of it).

Reply:
fixed.